src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -658,14 +659,15 @@ configurationRegistry.registerConfiguration({
 	title: nls.localize('notebookConfigurationTitle', "Notebook"),
 	type: 'object',
 	properties: {
-		[DisplayOrderKey]: {
-			description: nls.localize('notebook.displayOrder.description', "Priority list for output mime types"),
-			type: ['array'],
-			items: {
-				type: 'string'
-			},
-			default: []
-		},
+		// {{SQL CARBON EDIT}} Remove unused VS Code Notebook configurations
+		// [DisplayOrderKey]: {
+		// 	description: nls.localize('notebook.displayOrder.description', "Priority list for output mime types"),
+		// 	type: ['array'],
+		// 	items: {
+		// 		type: 'string'
+		// 	},
+		// 	default: []
+		// },
 		[CellToolbarLocation]: {
 			description: nls.localize('notebook.cellToolbarLocation.description', "Where the cell toolbar should be shown, or whether it should be hidden."),
```

Comment:
It would probably be good to add a note that this is added for compatibility reasons only but doesn't currently actually work so that customers aren't confused...

Reply:
Like in the actual user-facing description?

Reply:
Yes. I assume we don't have any plans currently of actually supporting this setting?

Reply:
Probably not. Have we done any kind of compatibility warning like that elsewhere, or should I throw something together?

Reply:
Nope, we typically just remove them. This is a pretty special case. Something like

`Where the cell toolbar should be shown, or whether it should be hidden. NOTE: This setting does not currently affect anything, it exists for compatibility reasons.`

(note for this you'll need to move it to this file and reference that one - https://github.com/microsoft/azuredatastudio/blob/050d2cc98f736206cb9417e07912275393be0cd8/src/sql/base/common/locConstants.ts due to how we pull in localized values from VS Code)